pkg/db-repo/repository.go

```diff
@@ -357,6 +371,132 @@ func ignoreCreatedAtIfNeeded(scope *gorm.Scope) {
 	}
 }
 
+// parses the []uint8 (from mysql) for field x, if there is a field xSet with type []string
+func GormParseMysqlSetType(scope *gorm.Scope) {
+	vo := reflect.ValueOf(scope.Value)
+	if vo.Kind() == reflect.Invalid {
+		return
+	}
+	for vo.Kind() == reflect.Ptr {
```

Comment:
This loop is now redundant if you move the slice handling to `parseMysqlSetType`